### ISSUE TITLE
Nutritional Liquifier Should Return Bottles

### ISSUE THREAD

**Issue by UnknownSerhan** [mekanism/Mekanism#8144]
### Issue description

The liquifier returns bowls but not bottles.

### Steps to reproduce

1. Put some honey bottles in the liquifier.
2. See that bottles are not returned.

### Minecraft version

1.21 (Latest)

### NeoForge version

21.0.40-beta

### Mekanism version

10.6.3 (Latest)

**Comment by Enderteck** [comment]
Bowls too then

**Comment by UnknownSerhan** [comment]
It already returns bowls

**Comment by kienerxyzzy** [comment]
> It already returns bowls

Did you know: #8177 exists and it still isn't marked as finished

**Comment by UnknownSerhan** [comment]
Yes and it is my issue as well. The thing is Mojang has weird spaghetti code and returns bowls of rabbit stews and bottles in a different way, and that’s why they are not working with the Mekanism implementation. So we can kind of call both of these return issues a “Mojank” issue but maybe we can get a special case fix from Mekanism
